File: TSPractice/StringProblems/p13.ts
//Get a Number of Vowels in a String

export let s1 = "Hello Midhun, welcome to Kerala.";

function countVowels(str: String): number {
    let vowelsCount = 0;
    for(let character of str.split("")){
        if(character == 'a' || character == 'e' || character == 'i' || character == 'o' || character == 'u') {
            vowelsCount++;
        }
    }
    return vowelsCount;
}

console.log(countVowels(s1));

File: Java/src/StringProblems/p13.java
//Get a Number of Vowels in a String

package StringProblems;

public class p13 {
    public static void main(String[] args) {
        String s1 = "Hello Midhun, welcome to Kerala.";
        System.out.println(getVowelCount(s1));
    }

    public static int getVowelCount(String str) {
        int vowelCount = 0;
        for(char charecter : str.toCharArray()) {
            if(charecter == 'a' || charecter == 'e' || charecter == 'i' || charecter == 'o' || charecter == 'u'){
                vowelCount++;
            }
        }
        return vowelCount;
    }
}
